app/src/main/res/layout/nav_header_navigation_drawer.xml

```diff
@@ -24,9 +24,10 @@
     android:paddingBottom="@dimen/activity_vertical_margin"
     android:theme="@style/ThemeOverlay.AppCompat.Dark">
 
-    <de.hdodenhof.circleimageview.CircleImageView
+    <com.google.android.material.imageview.ShapeableImageView
       android:layout_width="64dp"
       android:layout_height="64dp"
+      app:shapeAppearanceOverlay="@style/ShapeAppearanceOverlay.App.Circular"
```

Comment:
Should this also have elevation specified?

Reply:
No here elevation will not be added since we added it  only to the activities where shadow was visible in the mocks.